src/Provider/GeocodeEarth/GeocodeEarth.php

```diff
@@ -67,7 +70,11 @@ public function geocodeQuery(GeocodeQuery $query): Collection
             throw new UnsupportedOperation('The GeocodeEarth provider does not support IP addresses, only street addresses.');
         }
 
-        $url = sprintf(self::GEOCODE_ENDPOINT_URL, urlencode($address), $this->apiKey, $query->getLimit());
+        $url = $this->root.'/v'.$this->version.'/search'.'?'.http_build_query([
```

Comment:
Just personal preference but I prefer sprintf over complex concatenation 

```suggestion
		$url = sprintf(%s/v%s/search?%s, $this->root, $this->version, http_build_query([
```